scripts/tools/EXT_Exp_MetricExportData.rst

```diff
@@ -0,0 +1,103 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+    t=tags['$t']
+    T=t.upper()
+%>
+:orphan:
+
+.. _ZET_experimental_metric_export_data:
+
+==========================================
+Metrics Export Data Experimental Extension
+==========================================
+
+API
+----
+
+* Structures
+
+    * ${t}_metric_export_data_format_exp_t
+    * ${t}_metric_calculate_exp_desc_t
+
+* Functions
+
+    * ${t}MetricGroupGetExportDataExp
+    * ${x}DriverCalculateMetricExportDataExp
+
+Sample Code
+------------
+
+The following code shows how to export metrics raw data.
+
+.. parsed-literal::
+
+    // Get size for export data and count for data format
+    size_t exportDataSize = 0;
+    uint32_t exportDataFormatCount = 0;
+    ${t}MetricGroupGetExportDataExp (hMetricGroup, rawData, rawDataSize,
+                                &exportDataSize, &exportDataFormatCount, nullptr, nullptr);
+
+    // Allocate for export data and data format
+    uint8_t* pExportData = malloc(exportDataSize);
+    ${t}_metric_export_data_format_exp_t* pExportFormatData =
+                            malloc(sizeof(${t}_metric_export_data_format_exp_t) * exportDataFormatCount);
+
+    // Retrieve export data and data format
+    ${t}MetricGroupGetExportDataExp (hMetricGroup, rawData, rawDataSize,
+                        &exportDataSize, &exportDataFormatCount, pExportData, pExportFormatData);
+
+
+
+The following code shows how to perform machine independent calculation using the exported data.
```

Comment:
`machine independent` a bit ambiguous. How about saying something like ` .. how to perform metrics calculation of collected data, which can be done in a different system` ?

Reply:
Ok . Modified